docs/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This probably belongs to build so should be named build-wallets.md

Please add a link to this page on the sidebar ./website/sidebars.json